skbase/lookup/_lookup.py

```diff
@@ -475,11 +509,13 @@ def get_package_metadata(
     path: Optional[str] = None,
     recursive: bool = True,
     exclude_non_public_items: bool = True,
+    exclude_non_public_modules: bool = True,
```

Comment:
duplicate argument in two different spellings?

Reply:
ah, I see - deprecation.
I don't think we need deprecation at this stage (no users or dependencies), but as you prefer.

Unless, of course you are a user and need deprecation.

Reply:
I am a user, but don't need deprecation!

I'm fine removing the deprecation here.

Reply:
I don't need deprecation either, happy with the direct change.

Reply:
I've added a commit to remove the deprecation. That keeps life easy for now.